### PR TITLE
41 contact section validation

### LINKED ISSUE #41: Contact section validation

### PR DESCRIPTION
**Pull request by Roxha-V**
A test was made to test the functionality of the component Finaly the changes from the validation test were integrated into the real code with the already established styles.

### CODE REVIEW

**Comment by devPalter** [comment]
@Roxha-V para revisar los cambios necesito que el pr este apuntando a develop, podrias corregirlo y avisarme por telegram asi lo reviso c:

**Comment by Roxha-V** [comment]
@devPalta  listou

File: app/components/contact_section/index.jsx [hunk 1]
```diff
@@ -1,5 +1,51 @@
...
 "use client";
+
+import React, { useEffect, useState } from "react";
+
 function ContactComponent() {
+  const [formState, setFormState] = useState({
+    name: "",
+    message: "",
+  });
+  console.log(formState);
```
Comment (devPalter) [inline app/components/contact_section/index.jsx]: remove log

**Review by devPalter — CHANGES_REQUESTED** [review]

**Comment by Roxha-V** [comment]
problem solved - i suggest an interactive rebase to develop, so i unnecesary commits will be eliminated and the commit history will be more clear.

**Review by devPalter — APPROVED** [review]

**Pull request merged**